### PR TITLE
Conduction Resistance for coaxial HX

### PR DESCRIPTION
**Pull request by bo3mrh**
- Conduction resistance for coaxial heat exchanger is added
- IsFluidType function is no longer in used and replaced with IF statement.
- Add a function for Bertsch average HTC calculation.
- Update ACHP documentation.

### CODE REVIEW

File: ACHP/CoaxialHX.py [hunk 1]
```diff
@@ -20,6 +20,8 @@ def Update(self,**kwargs):
         self.A_r_wetted=pi*self.ID_i*self.L
         # Wetted area of the glycol side (not including outer tube)
         self.A_g_wetted=pi*self.OD_i*self.L
+        # total area of conduction
+        self.A_c_w=(pi/4) * (self.OD_i**2 - self.ID_i**2) * self.L
```
Comment (ibell) [inline ACHP/CoaxialHX.py]: Is this the right area to be used in the calculation for the resistance?  Shouldn't you use the radial thermal resistance with the ln(r1/r2) term?

Comment (bo3mrh) [inline ACHP/CoaxialHX.py]: Thank you for pointing this out, and check after me. I forget about the cylindrical coordinate thermal conduction.
It is updated now!

**Comment by bo3mrh** [comment]
Hello Ian,

I hope everything goes fine with you. I hope you are following the pull requests ?

**Comment by ibell** [comment]
Sorry I've been trying to do lots of things in parallel, and haven't had
the time to look at achp stuff in a while. Not sure when I will have the
time in the coming weeks, but I'll try

**Comment by ibell** [comment]
Sorry for the delay, looks good from what I saw.

**Comment by ibell** [comment]
Merged.

**Pull request merged**